components/core/src/clp_s/search/SearchUtils.hpp

```diff
@@ -37,12 +32,32 @@ LiteralType node_to_literal_type(NodeType type);
 bool double_as_int(double in, FilterOperation op, int64_t& out);
 
 /**
- * Performs a wildcard match of a string against a pattern
- * @param s the string to match
- * @param p the pattern to match against
- * @return true if s matches p, false otherwise
+ * Converts a KQL string column descriptor delimited by '.' into a list of tokens. The
```

Comment:
Do you think it's better to put those functions in a class (maybe `SearchUtils`) like `StringUtils`?